test/foundry/apps.t.sol

```diff
@@ -749,4 +750,187 @@ contract FleekTest is Test {
 
         assertEq(fleekContract.balanceOf(DEPLOYER), 1);
     }
+
+    function testAddAccessPoint() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+
+        assertEq(
+            fleekContract.getAccessPointJSON("https://fleek_cloned.xyz"),
+            '{"tokenId":0,"score":0,"nameVerified":false,"contentVerified":false,"owner":"0x7fa9385be102ac3eac297483dd6233d62b3e1496"}'
+        );
+    }
+
+    function testFailRemoveAccessPoint() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+        fleekContract.removeAccessPoint("https://fleek_cloned.xyz");
+        fleekContract.getAccessPointJSON("https://fleek_cloned.xyz");
+    }
+
+    function testIsAccessPointNameVerified() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+        assertFalse(fleekContract.isAccessPointNameVerified("https://fleek_cloned.xyz")); // is false now.
+
+        fleekContract.setAccessPointNameVerify("https://fleek_cloned.xyz", true);
```

Comment:
Just a note for after this PR: This made me realize we need to have a whitelisted controller list with only Fleek on it at first.  And only those addresses can enable verification since we can't have any controller they add mark it as verified.  I created a card with more details [here](https://app.shortcut.com/fleekxyz/story/338/add-controller-whitelist-for-verification).

After we have that logic, then we should update these tests to use a different address and test the allowed/rejected conditions.

Reply:
great point.